draft-ietf-quic-transport.md

```diff
@@ -1037,8 +1037,17 @@ Servers MUST drop other packets that contain unsupported versions.
 
 Packets with a supported version, or no version field, are matched to a
 connection using the connection ID or - for packets with zero-length connection
-IDs - the address tuple.  If the packet doesn't match an existing connection,
-the server continues below.
+IDs - the address tuple, with the following exception.
+
+A server that uses a non-zero-length connection ID SHOULD handle Initial packets
+that share the same address tuple, Source and Destination Connection IDs, but
+contain different first ClientHello messages as belonging to different
+connections, so that an attacker racing a spoofed Initial packet to the server
```

Comment:
```suggestion
connections. This might be used by a client to defend against attacks that race spoofed Initial packets with legitimate ones.  A server can treat every Initial packet containing a CRYPTO frame at an offset of 0 as potentially creating a new connection, discarding any packet that has the same Destination Connection ID and CRYPTO frame content as one that has been answered.
```

Reply:
Thank you for the suggestion. Applied.